azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -265,8 +264,8 @@ Resume-ClusterNode
 ```
 
 > [!NOTE]
-> To apply the Network ATC settings across the cluster, repeat steps 1 through 5, step 7, and step 9 for each node of the cluster.
+> To apply the Network ATC settings across the cluster, repeat steps 1 through 5 (skip deleting the virtual switch as it was renamed), step 7, and step 9 for each node of the cluster.
```

Comment:
@helmste - I don't think we need to add (skip deleting the virtual switch as it was renamed. Step 8 is not the only step that is being skipped. They are also skipping step 6. We can leave the note as is.

Reply:
@ronmiab - I do think we need to include this.  The reason is because the customers have already renamed their virtual switch on all their nodes.  If you don't add this step, they will assume they need to delete their virtual switch which defeats the purpose of renaming it.  The purpose of renaming the virtual switch is so Network ATC will assume managing the switch instead of having it create a new one which would further increase the customer's downtime needed to implement these changes.